addons/mrp/static/src/components/wo_list_view_dropdown/wo_list_view_dropdown.js

```diff
@@ -33,11 +35,25 @@ export class MOListViewDropdown extends Component {
     }
 
     get statusColor() {
+        if (this.isBlocked) {
+            return this.colorIcons[this.props.record.data.working_state];
+        }
         return this.colorIcons[this.state.state] || "";
```

Comment:
Don't you need empty string as fallback here too ?
```suggestion
        const state = this.isBlocked ? this.props.record.data.working_state : this.state.state;
        return this.colorIcons[state] || "";
```